testsuite/tests/typing-misc/exp_denom.ml

```diff
@@ -79,3 +79,24 @@ Line 1, characters 18-23:
 Error: The constant "foo" has type "string"
        but an expression was expected of type "int"
 |}]
+
+let _ : int = while false do () done
+
+[%%expect {|
+Line 1, characters 14-36:
+1 | let _ : int = while false do () done
+                  ^^^^^^^^^^^^^^^^^^^^^^
+Error: This while expression has type "unit"
```

Comment:
We want `inline code` quote here is since `while` is not part of the grammatical structure of the sentence.

Reply:
Implemented!